src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  const stringModified = sourceString
+    .split(/;\s*/)
+    .filter((item) => item.trim() !== '');
```

Comment:
You can chain the `.split` and `.filter` methods directly without the need for an intermediate variable `stringModified`. Also, ensure that the chained methods have the same indentation level for better readability.